docs/pages/reference/backends.mdx

```diff
@@ -573,8 +573,10 @@ teleport:
 
 ## S3
 
-S3 buckets can only be used as storage for the recorded sessions. S3 cannot
-store the audit log or the cluster state.
+S3 buckets can be used as storage for the recorded sessions and audit log. 
```

Comment:
Suggestion: change the heading to `S3 (Session Recordings)` and use this wording intead:

Teleport supports using S3 as a backend for both session recordings
and audit logs. S3 cannot be used as the cluster state backend.
This section covers the use of S3 as a session recording backend.
For information on using S3 for audit logs, see the [Athena](#Athena)
section.

Reply:
thanks @zmb3 , please see update